### DISCUSSION TITLE
b站网页加载失败

### DISCUSSION THREAD

**Discussion by lq-259** [1715173329/UnblockNeteaseMusic#467] [Q&A]
用二进制部署的，日志里没看到相关错误，issues搜索了一下也没发现相同的问题
![Screenshot_20220115-115920_MiXplorer](https://user-images.githubusercontent.com/60541200/149608064-8b5bd000-bc01-4e04-bc63-ae0aaa88c3de.png)
![Screenshot_20220115-115936_哔哩哔哩](https://user-images.githubusercontent.com/60541200/149608072-1b05c16c-7e1a-4b65-aa98-8db77f3d4e9d.png)
![Screenshot_20220115-115947_设置](https://user-images.githubusercontent.com/60541200/149608074-be61c8ab-11aa-4b20-9d9d-aadca46a0e72.png)

**Comment by pan93412** [comment]
試試看用 strict 模式： `node app.js -s ...`

**Reply by lq-259** [reply]
似乎没效果
![Screenshot_20220117-094457_哔哩哔哩](https://user-images.githubusercontent.com/60541200/149688372-d8bd32ab-21d1-415a-b082-64259136e387.png)
![Screenshot_20220117-094508_JuiceSSH](https://user-images.githubusercontent.com/60541200/149688386-8fffd541-d8f6-433c-9209-280b8245d828.png)

**Reply by 1715173329** [reply]
-p 8000:8001 试试

**Reply by lq-259** [reply]
pac这样填吗
![Screenshot_20220117-133515_JuiceSSH](https://user-images.githubusercontent.com/60541200/149713797-e59d0186-cbf4-472b-9197-136b60b66e09.png)
![Screenshot_20220117-133943_设置](https://user-images.githubusercontent.com/60541200/149713862-7228e0e7-7e4c-4a55-ae2e-b3b6c10444dd.png)

**Reply by 1715173329** [reply]
pac 8000
